app/services/xokul/detsis.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Xokul
+  module Detsis
+    module_function
```

Comment:
```suggestion
    extend self
```

tercih ederdim

Reply:
ikisi arasında etkin bir fark yok. projede (xokul'da dahil) genel olarak `module_function` kullandığımız için böyle bırakacağım. yine de öneri için teşekkürler :+1: